components/kyma-environment-broker/internal/broker/instance_create.go

```diff
@@ -49,6 +49,8 @@ type ProvisionEndpoint struct {
 	shootProject      string
 	shootDnsProviders gardener.DNSProvidersData
 
+	kubeconfigOrigin string
```

Comment:
This is representing a kyma dashboard URL ( per landscape ).
Why not calling it this `kymaDashboardBaseURL` instead ? I'm confused with `kubeconfigOrigin`